contracts/bridges/polygon/FxBaseChildTunnel.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// IFxMessageProcessor represents interface to process message
+interface IFxMessageProcessor {
```

Comment:
If we remove `FxBaseChildTunnel` from this file, then we can consider moving this interface definition to the top of  `PolygonBridgeReceiver.sol`.

Reply:
alternatively, we could track this file with Vendoza and then delete the unnecessary parts (but that deletion would at least be tracked/reproducible/updatable)

Reply:
replace `FxBaseChildTunnel` with `FxChild` contract; moved to a `bridges/vendor` directory and started tracking with Vendoza

Reply:
and deleted all the unnecessary bits